pkg/webreg/webreg.go

```diff
@@ -611,37 +611,74 @@ Example:
-image:
+In order to use an image from one repository in the tests of another, it is necessary
+to first publish the image to the integration stream from the producer repository and
+import it in the consumer repository.
 </p>
 
-Both examples assume we are starting off with this ci-operator config:
-{{ yamlSyntax (index . "imageExampleBaseConfig") }}
+<h4 id="image-publication">Publishing an Image For Reuse</h3>
+<p>
+When configuring <code>ci-operator</code> for a repository, the <code>promotion</code>
+stanza declares which container images are promoted and where they are pushed. By
+default, all container images declared in the <code>images</code> block of a configuration
+are published. Promotion can be configured to exclude any of those default images as well
+as including other images, as well. In the following <code>ci-operator</code> configuration,
+the following images are promoted for reuse by other repositories:
+</p>
+<ul>
+  <li>the <code>:src</code> tag, published as <code>:repo-scripts</code> containing the latest version of the repository to allow for executing helper scripts</li>
+  <li>the <code>:test-bin</code> tag, published as <code>:repo-tests</code> containing built test binaries to allow for running the repository's tests</li>
+  <li>the <code>:component</code> tag, published as <code>:component</code> containing the component itself to allow for deployments and installations in end-to-end scenarios</li>
+</ul>
+
+<code>ci-operator</code> config:
+{{ yamlSyntax (index . "imagePromotionConfig") }}
+
+<h4 id="image-consumption">Consuming an Image</h3>
+<p>
+Once a repository is publishing an image for reuse by others, downstream users can
+configure <code>ci-operator</code> to use that image in tests by including it as a
+<code>base_image</code> or as part of the <code>tag_specification</code>. In general,
+images will be available as part of the <code>tag_specification</code> and explicitly
+including them as a <code>base_image</code> will only be necessary if the promoting
+repository is exposing them to a non-standard <code>ImageStream</code>. The following
+<code>ci-operator</code> imports a number of images:
+</p>
+<ul>
+  <li>the <code>:custom-scripts</code> tag, published as <code>myregistry.com/project/custom-scripts:latest</code></li>
+  <li>the <code>:component</code> and <code>:repo-{scripts|tests}</code> tags, by virtue of them being published under <code>ocp/4.4</code> and brought in with the <code>tag_specification</code></li>
+</ul>
+
+<code>ci-operator</code> config:
+{{ yamlSyntax (index . "imageConsumptionConfig") }}
+
+<p>
+Once the image has been configured to be an input for the repository's tests in the
+<code>ci-operator</code> configuration, either explicitly as a <code>base_image</code>
+or implicitly as part of the <code>tag_specification</code>, it can be used in tests
+in one of two ways. A registry reference step can be written to execute the shared tests
```

Comment:
make it clearer how you refer to the imported images.  they appear in the "stable" imagestream, right?  but also the env var mechanism of referencing them....